### DISCUSSION TITLE
Meeting Search radius displays in Miles (in Australia we use Kms)

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1162] [Q&A] [answered]
Can I customise the search results meeting distance metric from miles to kilometres? In Australia we use kilometres, so displaying miles will confuse/annoy our users when looking for a meeting close to their location.

Review staging site for testing: [(https://staging.al-anon.org.au/find-a-meeting/)] 

Attached is a screenshot, as it's only when searching that miles appears. 

<img width="1091" alt="Screen Shot 2023-06-27 at 10 26 27 am" src="https://github.com/code4recovery/12-step-meeting-list/assets/23748349/2cc0b627-5560-4748-bc64-c0cced769c2d">

**Comment by joshreisner** [comment, marked as answer]
yes, of course! you can change it to kms at Meetings > Settings

**Reply by nataliefaith2** [reply]
I have this selected but it's showing miles in the results for nearby meetings. 
<img width="929" alt="Screen Shot 2023-07-10 at 8 36 03 pm" src="https://github.com/code4recovery/12-step-meeting-list/assets/23748349/e8496de3-76d7-4c7c-bba7-30dcf277428a">

And it displays kms everywhere else.

**Reply by joshreisner** [reply]
ok i think i see what's happening. typically we expect the 12 Step Meeting List users to use the built-in meetings page at `/meetings` - in your case that would be https://staging.al-anon.org.au/meetings/ -- i see that one has an error which you could resolve by going to WordPress Settings and setting the timezone there -- that page will have the configuration setting that you used above.

Looks like you may have embedded TSML UI manually on the `/find-a-meeting` page which you can also do, but when you embed manually you need to provide the settings yourself. Here is how you add the KMs setting: https://github.com/code4recovery/tsml-ui#use-kilometers

One other reason to use the standard page though: it provides the Al-Anon meeting types to TSML UI, otherwise you have to do that manually which is a pain
